lib/tasks/javascript.rake

```diff
@@ -0,0 +1,20 @@
+TIMEZONES_DEFINITIONS = 'https://raw.githubusercontent.com/moment/moment-timezone/develop/data/meta/latest.json'
+
+task 'javascript:update_constants' => :environment do
+  require 'holidays'
+  holiday_regions = Holidays.available_regions.map(&:to_s) - ["ecbtarget", "federalreserve", "federalreservebanks", "fedex", "nerc", "unitednations", "ups"]
```

Comment:
I would extract this array in a constant with a helpful name (like `UNUSED_REGIONS`)